app/assets/javascripts/user-satisfaction-survey.js

```diff
@@ -67,12 +67,21 @@
     setSurveyUrl: function(href) {
       var $surveyLink = $('#take-survey');
       var surveyUrl = $('#user-satisfaction-survey-container').data('survey-url');
+      var surveyStarts = new Date(2016, 1, 26).getTime();
+      var surveyEnds = new Date(2016, 1, 27, 23, 59, 59).getTime();
+
+      if (userSatisfaction.currentDate() >= surveyStarts &&
+          userSatisfaction.currentDate() <= surveyEnds) {
+        surveyUrl = 'https://www.surveymonkey.co.uk/r/2MRDLTW';
+      }
```

Comment:
That would mean under normal circumstances it isn't configurable by consumers. We did discuss this briefly offline earlier and the recommendation was to take this route.

I guess given the transient nature of this particular survey these things haven't been given too much consideration. Shrug.